### PR TITLE
Code quality issues

### PR DESCRIPTION
**Pull request by 4LineTime**
Modified style, spelling and grammar issues in convert_state.py

### CODE REVIEW

**Comment by claraj** [comment]
@ajgoranson can you review?

**Comment by ajgoranson** [comment]
Dang This looks good! I would like your use of variable names

File: convert_state.py [hunk 1]
```diff
@@ -1,60 +1,67 @@
...
+
+        choice = input('Enter choice: ')
+
+        if choice == '1':
+            convert_state_to_abbreviation(state_name_dict)
+
+        elif choice == '2':
+            convert_abbreviation_to_state(state_abbr_dict)
+         
+        elif choice ==  '3':
```
Comment (claraj) [inline convert_state.py]: Single space after ==

File: convert_state.py [hunk 2]
```diff
@@ -1,60 +1,67 @@
...
-    userInput   = input("Enter abbrviation name").upper()
-    result = dictionary.get(userInput)
-    if result == None:
-        print(' abreviation not found')
+            print('Try again.')
+
+
+def convert_state_to_abbreviation(dictionary):
+    """ Convert state to abbreviation  """
+    user_input = input('Enter state name ').capitalize()   # User inputs state name
```
Comment (claraj) [inline convert_state.py]: This comment isn't needed

**Review by claraj — CHANGES_REQUESTED** [review]
Looks much better! A few small issues to look at.

File: convert_state.py [hunk 3]
```diff
@@ -1,60 +1,67 @@
...
+    """ Convert state to abbreviation  """
+    user_input = input('Enter state name ').capitalize()
+    result = dictionary.get(user_input)
+
+    if result is None:
+        print('State not found')
+
     else:
-        print("the state with the abbreviation   " + userInput + " is " + result)
+        print('The abbreviation for ' +  user_input + ' is ' + result)
```
Comment (claraj) [inline convert_state.py:50]: use f-string

**Review by claraj — COMMENTED** [review]
couple of small comments but looks much better. thanks!

**Pull request closed without merging**